artview/components/nav.py

```diff
@@ -194,54 +194,62 @@ def createHelpUI(self):
     #   Update Methods   #
     ######################
 
-    def _update_InfoUI(self):
-        '''Update the info label.'''
-        self.dirIn = os.path.dirname(self.Vradar.value.filename)
-        self.infodir.setText("Directory: %s"%(self.dirIn))
-        self.infofile.setText("File: %s"%(
-            os.path.basename(self.Vradar.value.filename)))
+#    def _update_InfoUI(self):
+#        '''Update the info label.'''
+#        self.dirIn = os.path.dirname(self.Vradar.value.filename)
+#        self.infodir.setText("Directory: %s"%(self.dirIn))
+#        self.infofile.setText("File: %s"%(
+#            os.path.basename(self.Vradar.value.filename)))
 
     def _update_tools(self):
-        '''Update the info label.'''
```

Comment:
This function is now much bigger, since I need to add test to handle special cases (empty dir, first file in dir etc.)